apps/web/playwright/booking/utils/bookingUtils.ts

```diff
@@ -0,0 +1,250 @@
+import { expect, type Page } from "@playwright/test";
+
+import type { Fixtures } from "@calcom/web/playwright/lib/fixtures";
+
+const EMAIL = "test@test.com";
+const EMAIL2 = "test2@test.com";
+const PHONE = "+55 (32) 983289947";
+const scheduleSuccessfullyText = "This meeting is scheduled";
+const reschedulePlaceholderText = "Let others know why you need to reschedule";
+
+interface QuestionActions {
+  [key: string]: () => Promise<void>;
+}
+
+type BookingOptions = {
+  hasPlaceholder?: boolean;
+  isReschedule?: boolean;
+  isRequired?: boolean;
+};
+
+type customLocators = {
+  shouldChangeSelectLocator: boolean;
+  shouldUseLastRadioGroupLocator: boolean;
+  shouldUseFirstRadioGroupLocator: boolean;
+  shouldChangeMultiSelectLocator: boolean;
+};
+
+export const loginUser = async (page: Page, users: Fixtures["users"]) => {
```

Comment:
I believe this would be better located in a before each step like shown here: apps/web/playwright/auth/auth-index.e2e.ts

Reply:
also page.goto("/event-types"); can be located inside the beforeEach hook directly instead of inside of the login function